assets/js/edit.js

```diff
@@ -261,6 +260,13 @@ jQuery(document).ready(function($){
                 }
             });
         }
+
+        if (['html', 'section_break'].indexOf(select.val()) >= 0) {
+            var label = parent.find('.field-label');
```

Comment:
@iamntz Small point, but I prefer that a variable that holds a `jQuery` instance be prefixed with `$`. Not a blocker, edit.js does not follow this convention well.